tests/benchdnn/dnnl_common.cpp

```diff
@@ -1132,9 +1132,10 @@ int check_total_size(res_t *res, dnnl_primitive_t prim_ref) {
     // 0.75f is taken randomly and is subject to change in future.
     const double capacity_factor = 0.75;
     const double benchdnn_device_limit = capacity_factor * device_max_capacity;
+    // Note: there used to be a separate limit for combined memory pool, however
+    // it didn't work even at 0.80 point due to, likely, system memory
+    // requirements. Use same limit for combined cpu and pure cpu cases.
     const double benchdnn_cpu_limit = capacity_factor * cpu_device_capacity;
-    // Note: used to be 0.90 until large f64 conv cases on Xe2-LPG emerged.
-    const double benchdnn_combined_limit = 0.80 * cpu_device_capacity;
```

Comment:
Rather than just constantly reducing the capacity (and consequently skipping problems more aggressively), would it be reasonable to do a form of lazy evalutation? Essentially what I am thinking is
```
    if(limit > 0.95*capacity) {
            return skip problem; // Same logic as currently
    } else {
          auto ret = try_execute();
          if(ret == out_of_memory && limit > 0.8*capacity)
                return skip_problem;
          return ret;
    }
```